zebra-chain/Cargo.toml

```diff
@@ -109,7 +111,7 @@ rand_chacha = { version = "0.3.1", optional = true }
 
 tokio = { version = "1.28.2", features = ["tracing"], optional = true }
 
-zebra-test = { path = "../zebra-test/", optional = true }
+zebra-test = { path = "../zebra-test/", version = "1.0.0-beta.25", optional = true }
```

Comment:
Do the dev-dependencies need versions as well?
This applies to all Zebra crates.

Reply:
Looks like the dev-dependencies do not need versions